Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -252,6 +252,17 @@ theorem isSuccPrelimit_iff_succ_lt : IsSuccPrelimit b ↔ ∀ a < b, succ a < b
 @[deprecated isSuccPrelimit_iff_succ_lt (since := "2024-09-05")]
 alias isSuccLimit_iff_succ_lt := isSuccPrelimit_iff_succ_lt
 
+theorem coe_succ_Iio {a : α} (h : IsSuccPrelimit a)
+    (x : Iio a) : (succ x).1 = succ x.1 := by
```

Comment:
It would be more readable as
```suggestion
theorem coe_succ_Iio {a : α} (h : IsSuccPrelimit a) (x : Iio a) : (succ x).1 = succ x.1 := by
```
(preferred) or
```suggestion
theorem coe_succ_Iio {a : α} (h : IsSuccPrelimit a) (x : Iio a) :
    (succ x).1 = succ x.1 := by
```
(okay, but unnecessarily space-taking)

Same below